kernel-python/declarativewidgets/util/serializers.py

```diff
@@ -28,6 +29,22 @@
 else:
     from .base_serializer_py3 import BaseSerializer
 
+def serialize_pandas_df(obj, **kwargs):
```

Comment:
When I mentioned that the Spark serializer should delegate to the Pandas Serializer, I meant for it to call `PandasSeriesSerializer.serialize()` not to refactor the method as top level. These are all static methods, so there was no need to move outside the class.